src/components/setAPIsTemplate.tsx

```
@@ -0,0 +1,91 @@
+
+interface Props {
+  temps: TemplateAPI[];
+  setTemps: (temps: TemplateAPI[]) => void;
+  label: string;
+  endpoint: string;
+  APIkey: string;
+}
+export function SetAPIsTemplate({
+  endpoint,
+  APIkey,
+  temps: temps,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)